awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHostList.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
This is just a file rename.  I believe it was called InventoryHosts, or Hosts. No code has changed here.